packages/expo-av/src/Video.types.ts

```diff
@@ -247,6 +247,10 @@ export type VideoProps = {
    * @hidden
    */
   rotation?: number;
+  /**
+   * @hidden
+   */
+  videoStyle?: StyleProp<ViewStyle>;
```

Comment:
```suggestion
  /**
   * @hidden
   */
  videoStyle?: StyleProp<ViewStyle>;
```

Let's move this prop outside of React Native specific props to the custom ones an bit above, remove `@hidden` annotation add the correct description (see `posterStyle` for the example).